src/dom-to-image-more.js

```diff
@@ -1199,13 +1199,31 @@
                 sandbox.style.visibility = 'hidden';
                 sandbox.style.position = 'fixed';
                 document.body.appendChild(sandbox);
-                // Ensure that the iframe is rendered in standard mode
+                sandbox.id = sandbox.contentDocument.title = 'domtoimage-sandbox';
+                // Ensure the iframe's character rendering matches the document (with UTF-8 fallback).
                 const charset = document.createElement('meta');
                 charset.setAttribute('charset', document.characterSet || 'UTF-8');
                 sandbox.contentDocument.head.appendChild(charset);
-                sandbox.contentDocument.title = 'sandbox';
+                // Ensure the document is rendered in standard mode by coercing doctype.
+                // If the parent document has a Trusted Types policy, fallback to quirks mode.
+                let isTrustedTypesRequired;
+                try {
+                    const xhr = new XMLHttpRequest();
```

Comment:
Is there no better way to determine this _for the currently loaded page_ without reloading it this way? Seems like this will be very slow generating an entire round-trip + server response time...

Reply:
We may have to let people opt-in to this...

Reply:
I think it wouldn't be as bad if it was fetching the `HEAD` only instead of `GET`.  
Even with the artificial 2000ms delay that "Slow 3G" throttling adds, its a consistent 100ms.

Reply:
Resolved in 6b609d3